Supporting Resources/v1.5/MarkDown/SAMM-1.5.1.md

```diff
@@ -457,7 +457,7 @@ TODO change activity B of LSM1
 | | LSM1 | LSM2 | LSM3 |
 |:-------------|:------------- |:---------------|:-------------|
 |**Objective** | **Awareness of potential attack patterns.** | **Knowledge about impact of attack patterns to released hardware and software versions and variants.** | **Minimize negative impacts for customers.**|
- **Activities** | **A.** Maintenance of attack pattern catalogue **B.** Maintenance of attack pattern catalogue| **A.** Implement all relevant penetration tests on all relevant hardware and software versions and variants **B.** Execute all relevant penetration tests on all relevant hardware and software | **A.** Deploy degradations to customer **B.** Deploy updates to customer |
+ **Activities** | **A.** Maintenance of attack pattern catalogue | **A.** Implement all relevant penetration tests on all relevant hardware and software versions and variants **B.** Execute all relevant penetration tests on all relevant hardware and software | **A.** Deploy degradations to customer **B.** Deploy updates to customer |
```

Comment:
Activities: A. --> Where there is an A activity, there also must be a B Activity for LSM1.

Reply:
should be fixed with d7487364fe471c53230396fbc80f3a5b6ec25c08